contracts/src/NativeTokenSource.sol

```diff
@@ -40,8 +40,16 @@ contract NativeTokenSource is INativeTokenBridge, TeleporterTokenSource {
     IWrappedNativeToken public immutable wrappedToken;
 
     /**
-     * @notice Initializes this source token bridge instance
-     * @dev Teleporter fees are paid by a {IWrappedNativeToken} instance.
+     * @notice Initializes this source token bridge instance to send
+     * native tokens to the specified destination chain and token bridge instance.
```

Comment:
```suggestion
     * @notice Initializes this token source bridge instance to send native tokens to other chains.
```
I think this is a bit misleading so no destination bridge chain/instance is specified in the constructor.

Reply:
updated